diff --git a/src/Make_cyg_ming.mak b/src/Make_cyg_ming.mak
index 0f6a492..70dfbe6 100644
--- a/src/Make_cyg_ming.mak
+++ b/src/Make_cyg_ming.mak
@@ -867,7 +873,7 @@ GvimExt/gvimext.dll: GvimExt/gvimext.cpp GvimExt/gvimext.rc GvimExt/gvimext.h
 	$(MAKE) -C GvimExt -f Make_ming.mak CROSS=$(CROSS) CROSS_COMPILE=$(CROSS_COMPILE) CXX='$(CXX)' STATIC_STDCPLUS=$(STATIC_STDCPLUS)
 
 libvterm/.libs/libvterm.a :
-	cd libvterm && $(MAKE) libvterm.la
+	cd libvterm && $(MAKE) libvterm.la CC='$(CC)'
 
 clean:
 	-$(DEL) $(OUTDIR)$(DIRSLASH)*.o
